src/hydra-queue-runner/build-remote.cc

```diff
@@ -370,12 +371,15 @@ void State::buildRemote(ref<Store> destStore,
                     break;
                 default:
                     result.stepStatus = bsAborted;
+                    result.errorMsg += " {default/abort}";
                     break;
             }
-            if (result.stepStatus != bsSuccess) return;
-        }
 
-        result.errorMsg = "";
+            if (result.stepStatus != bsSuccess) {
+                result.errorMsg += " Log: " + chomp(readFile(result.logFile));
```

Comment:
We definitely should not store the log file in the database. It's already way to big.

Reply:
Good point.  Is it worth storing the tail end of the log (10 lines?) for informative purposes, or would you rather there was no log message information stored?

Reply:
I investigated this further and have updated this PR.  There are two locations where this diagnostic information could be viewed: (1) on the hydra system directly via `journalctl`, and (2) on the `.../build/NUM#tabs-buildsteps` page.  I had been focused on (1), and hadn't considered (2).  For (2) there is already a mechanism for viewing the logfile, so I've updated to only show the logfile in (1) and not add this information to the database.